web-server/src/components/Service/SystemStatus.tsx

```diff
@@ -0,0 +1,174 @@
+    addPage({
+      page: {
+        ui: 'system_logs',
+        title: `${serviceTitle[serviceName]} Logs`,
+        props: { serviceName }
+      }
+    });
+  };
+
+  return (
+    <FlexBox col gap={2} sx={{ padding: '16px' }}>
+      <Line bold white fontSize="24px" sx={{ mb: 2 }}>
+        System Status
+      </Line>
+      <Divider sx={{ mb: 2, backgroundColor: 'rgba(255, 255, 255, 0.2)' }} />
+
+      {loading ? (
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center'
+          }}
+        >
+          <CircularProgress color="primary" size={50} />
+        </Box>
+      ) : (
+        <FlexBox col gap={2}>
+          {Object.keys(services).map((serviceName) => {
+            const serviceKey = serviceName as ServiceNames;
+            const { isUp } = services[serviceKey];
+            const borderColor = serviceColor[serviceKey];
+
+            return (
+              <CardRoot
+                key={serviceName}
+                onClick={() => handleCardClick(serviceKey)}
+                sx={{
+                  transition: 'box-shadow 0.2s ease',
+                  '&:hover': {
+                    boxShadow: '0 4px 15px rgba(0, 0, 0, 0.1)'
+                  },
+                  backgroundColor: 'rgba(255, 255, 255, 0.05)',
+                  borderRadius: '12px',
+                  border: `1px solid ${
+                    isUp ? borderColor : alpha(theme.colors.error.main, 0.3)
+                  }`,
+                  padding: '16px',
+                  cursor: 'pointer',
+                  position: 'relative',
+                  overflow: 'hidden'
+                }}
+              >
+                <FlexBox col flexGrow={1} minHeight="5em">
+                  <FlexBox alignCenter justifyContent="space-between">
+                    <Line
+                      white
+                      bold
+                      sx={{
+                        fontSize: '1.2em',
+                        display: 'flex',
+                        alignItems: 'center'
+                      }}
+                    >
+                      {serviceTitle[serviceKey]}
+                      <Box
+                        component="span"
+                        sx={{
+                          display: 'inline-block',
+                          marginLeft: '6px',
+                          width: '10px',
+                          height: '10px',
+                          borderRadius: '50%',
+                          backgroundColor: isUp
+                            ? theme.colors.success.main
+                            : theme.colors.error.main
+                        }}
+                      />
+                    </Line>
+                  </FlexBox>
+
+                  <FlexBox col relative fullWidth flexGrow={1}>
+                    <FlexBox
+                      alignCenter
+                      sx={{ width: '100%', paddingTop: '8px' }}
```

Comment:
rather then giving `width:100%` use `fullwidth` prop on `Flexbox`
also you can use `paddingTop={1}` directly on `Flexbox`